docs/source/llms/tracing/tracing-schema.rst

```diff
@@ -0,0 +1,328 @@
+                </p>
+            </div>
+            <div class="image-box">
+                <img src="../../_static/images/llms/tracing/schema/trace_architecture.png"/>
+            </div>
+        </div>
+
+.. |trace-info| raw:: html
+
+        <div class=""main-container"">
+            <div>
+              <h4>Trace Info Structure</h4>
+              <p> The Trace Info within MLflow's tracing feature aims to provide a lightweight snapshot of critical data about the overall trace. 
+                This includes the logistical information about the trace, such as the experiment_id, providing the storage location for the trace, 
+                as well as trace-level data such as start time and total execution time. The Trace Info also includes tags and status information for 
+                the trace as a whole.
+              </p>
+            </div>
+            <div class="image-box">
+                <img src="../../_static/images/llms/tracing/schema/trace_info_architecture.png"/>
+            </div>
+        </div>
+
+.. |trace-data| raw:: html
+
+        <div class=""main-container"">
+            <div>
+              <h4>Trace Data Structure</h4>
+              <p> The Trace Data within MLflow's tracing feature provides the core of the trace information. Within this object is a list of 
+                <a href="../../python_api/mlflow.entities.html#mlflow.entities.Span">Span</a> objects that represent the individual steps of the trace. 
+                These spans are associated with one another in a hierarchical relationship, providing a clear order-of-operations linkage of what 
+                happened within your application during the trace.
+              </p>
+            </div>
+            <div class="image-box">
+                <img src="../../_static/images/llms/tracing/schema/trace_data_architecture.png"/>
+            </div>
+        </div>
+
+.. |span-architecture| raw:: html
+
+        <div class=""main-container"">
+            <div>
+              <h4>Span Structure</h4>
+              <p> The Span object within MLflow's tracing feature provides detailed information about the individual steps of the trace. 
+                Each Span object contains information about the step being instrumented, including the span_id, name, start_time, parent_id, status, 
+                inputs, outputs, attributes, and events.
+              </p>
+            </div>
+            <div class="image-box">
+                <img src="../../_static/images/llms/tracing/schema/span_architecture.png"/>
+            </div>
+        </div>
+
+
+.. container:: tracing-responsive-tabs
+
+    .. tabs::
+
+        .. tab:: Trace Architecture
+
+            |trace-architecture|
+
+        .. tab:: Trace Info
+
+            |trace-info|
+
+        .. tab:: Trace Data
+
+            |trace-data|
+
+        .. tab:: Span Architecture
+
+            |span-architecture|
+
+
+Trace Schema
```

Comment:
Will it be messy if we convert these sections as tables, probably one for each high-level object (TraceInfo / TraceData / Span)? The rational is that the h2 and h3 is hardly distinguishable in Sphinx, so it is not super easy to grasp the data structure from flat text sections. WDYT?

Reply:
Makes sense. It's a bit more dense, but I'm going with the table approach - it's a bit more concise